temporalcli/commandsgen/commands.yml

```
@@ -394,6 +394,160 @@ commands:
         description: Reason for terminating the batch job.
         required: true
 
+  - name: temporal deployment
+    summary: Describe, list, and operate on Deployments
+    description: |
+      +---------------------------------------------------------------------+
+      | CAUTION: Worker Deployment is experimental. Deployment commands are |
+      | subject to change.                                                  |
+      +---------------------------------------------------------------------+
+
+      Deployment commands perform operations on Worker Deployments:
+
+      ```
+      temporal deployment [command] [options]
+      ```
+
+      For example:
+
+      ```
+      temporal deployment list
+      ```
+    option-sets:
+      - client
+    docs:
+      description-header: >-
+        Temporal Deployment commands enable operations on Worker Deployments,
+        such as describe, list, update-current, and get-current, simplifying
+        versioning and management of workers.
+      keywords:
+        - deployment
+        - deployment describe
+        - deployment list
+        - deployment get-current
+        - deployment update-current
+
+  - name: temporal deployment describe
+    summary: Show properties of a Deployment
+    description: |
+      +---------------------------------------------------------------------+
+      | CAUTION: Worker Deployment is experimental. Deployment commands are |
+      | subject to change.                                                  |
+      +---------------------------------------------------------------------+
+
+      Describes properties of a Deployment, such as whether it is current,
+      some custom metadata, or a list of its task queues.
```

If we always return all three of these things, the word "or" is misleading.
```suggestion
      Describes properties of a Deployment, such as whether it is current,
      the list of its task queues, custom metadata if present, and reachability
      status if requested.
```

---

We don't print empty things, e.g., an empty list of task queues, but I guess the meaning is implicit. I prefer your wording though...